### PR TITLE
Change matching pattern on settings.gradle

### PR DESCRIPTION
**Pull request by yarivkenan**
Link doesn't work on f8app,
as it is, android projects don't have to contain the matching string and then the linking would not work, I've switched it to the much simpler new-line.
If you can find a better common minimum string for settings.gradle please do.

Commit Msg:
Current string, `include ':app'\n`, is not a minimum common string.
Instead a simple `\n` will do.

### CODE REVIEW

**Comment by grabbou** [comment]
@Kureev Mind taking a look?

**Comment by Kureev** [comment]
I just scaffolded a new `react-native create` app. Line 3 of settings.gradle file in android folder: `include :app`. Not sure if we have any plans to support various custom configurations. If you want to aim to process *any* settings.gradle file possible, most probably you end up with just adding new lines to the end of file (that's the safest way).

tl;dr: don't think this patch is required, but don't think it break anything as well.

**Comment by yarivkenan** [comment]
Hi @Kureev , thanks for answering.
It doesn't work for facebook feature app [f8app](https://github.com/fbsamples/f8app/blob/master/android/settings.gradle)
And might cause confusion.
Other then that you're right and can close if you want.

**Comment by Kureev** [comment]
No, I think you're right. `\n` should do the trick.

**Comment by Kureev** [comment]
@facebook-github-bot shipit
